test/lib/y2caasp/clients/admin_role_dialog_test.rb

```diff
@@ -21,40 +22,7 @@
       allow(Yast::LanItems).to receive(:dhcp_ntp_servers).and_return({})
     end
 
-    it "creates wizard dialog in normal mode" do
-      allow(Yast::Mode).to receive(:normal).and_return(true)
-
-      expect(Yast::Wizard).to receive(:CreateDialog)
-
-      subject.run
-    end
-
-    it "closed wizard dialog in normal mode" do
-      allow(Yast::Mode).to receive(:normal).and_return(true)
-
-      expect(Yast::Wizard).to receive(:CloseDialog)
-
-      subject.run
-    end
-
-    it "shows CWM widgets" do
-      allow(Yast::Mode).to receive(:normal).and_return(true)
-
-      expect(Yast::CWM).to receive(:show).and_return(:next)
-
-      subject.run
-    end
-
-    context "when some NTP server is detected via DHCP" do
-      let(:ntp_servers) { ["ntp.example.com"] }
-
-      it "proposes to use it by default" do
-        expect(Yast::LanItems).to receive(:dhcp_ntp_servers).and_return("eth0" => ntp_servers)
-        expect(Y2Caasp::Widgets::NtpServer).to receive(:new)
-          .with(ntp_servers).and_call_original
-        subject.run
-      end
-    end
-
+    include_examples "displays the dialog"
+    include_examples "NTP from DHCP"
```

Comment:
why not use also CWM dialog shared examples? https://github.com/yast/yast-yast2/blob/master/library/cwm/src/lib/cwm/rspec.rb#L153